app/helpers/patients_helper.rb

```diff
@@ -67,10 +67,10 @@ def disable_continue?(patient)
     patient.pledge_info_present? ? 'disabled="disabled"' : ''
   end
 
-  def dcaf_pledge_limit_help_text
-    first_tri = ENV['DCAF_PLEDGE_LIMIT_FIRST_TRI'] || 100
-    second_tri = ENV['DCAF_PLEDGE_LIMIT_SECOND_TRI'] || 300
-    later_care = ENV['DCAF_PLEDGE_LIMIT_LATER_CARE'] || 600
+  def pledge_limit_help_text
+    first_tri = ENV['PLEDGE_LIMIT_FIRST_TRI'] || 100
```

Comment:
should swap these env vars over to `DARIA_PLEDGE_LIMIT_xxx` in keeping with the naming scheme